yeat/cli/cli.py

```diff
@@ -72,37 +76,24 @@ def options(parser):
         default=1,
         help="execute workflow with T threads; by default T=1",
     )
-    parser.add_argument("-v", "--version", action="version", version=f"YEAT v{yeat.__version__}")
-
-
-def read_input_types(parser):
-    read_input_options = parser.add_argument_group("input read options")
-    mx = read_input_options.add_mutually_exclusive_group(required=True)
-    mx.add_argument(
-        "--paired",
-        metavar=("READ1", "READ2"),
-        type=str,
-        nargs=2,
-        help="paired-end reads in FASTQ format",
+    parser.add_argument(
+        "-n",
+        "--dry-run",
+        action="store_true",
+        help="construct workflow DAG and print a summary but do not execute",
     )
-    mx.add_argument("--pacbio", metavar="READ", type=str, help="PacBio HiFi-reads in FASTQ format")
```

Comment:
No more need for this function with the new config file. In the config file, I expect users to provide the correct number of reads/files for each assembly algorithm. 

In future PR, I would like to do some input checking, ensuring that there are enough files, but for now, I would like to "trust" that the user knows what they are doing.